### PR TITLE
feat: update flat model and demo data

### PR DESCRIPTION
**Pull request by BrendaHijmans**
What are the main changes you did:
- see diff for data model and ontology changes
-demo data update, e.g.:
-organisations lead and additional split
-further migrated data updates

how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] update ssr
- [ ] fix failing tests (something with collection types)

### CODE REVIEW

File: data/_demodata/applications/datacatalogue/Mappings.csv [hunk 1]
```diff
@@ -1 +0,0 @@
-source,source version,target,target version,cdms other,mapping status,ETL frequency,ETL specification url,ETL specification document
```
Comment (mswertz) [inline data/_demodata/applications/datacatalogue/Mappings.csv:1]: this file should be deleted?

File: data/_ontologies/Collection types FLAT.csv [hunk 2]
```diff
@@ -2,7 +2,7 @@ name,label,parent,codesystem,code,ontologyTermURI,definition
 Biobank,,,,,,
 Clinical trial,,,,,,
 Clinical study,,,,,,what is difference trial?
-Cohort,,,,,,to be deleted
+Cohort,,,,,,
```
Comment (mswertz) [inline data/_ontologies/Collection types FLAT.csv:5]: delete?

**Review by mswertz — CHANGES_REQUESTED** [review]
two minor points

File: data/_demodata/applications/datacatalogue/Mappings.csv (hunk 1 above)
Comment (BrendaHijmans) [reply data/_demodata/applications/datacatalogue/Mappings.csv:1]: It is already deleted.... strange 
image

**Comment by mswertz** [comment]
oh wait, testing failing

**Pull request merged**